### DISCUSSION TITLE
Svelte flow export as custom element

### DISCUSSION THREAD

**Discussion by fcollonval** [xyflow/xyflow#3690] [New Features]
Hey
Thanks a lot for this awesome library. I saw with great interest the new integration in Svelte. Do you plan to use the feature to export it as custom element to allow its integration in more frameworks?

Xref https://svelte.dev/docs/custom-elements-api

**Comment by moklick** [comment]
It's a good idea. We will look into this when we have a stable Svelte Flow version :)
